Library/Homebrew/utils/notability.rb

```diff
@@ -25,11 +31,19 @@ def github(user, repo)
     "GitHub repository too new (<30 days old)"
   end
 
-  def gitlab(user, repo)
+  def gitlab_repo_data(user, repo)
+    return @gitlab_data unless @gitlab_data.nil?
```

Comment:
Need to do something similar to `@github_repo_data` above or it'll reuse the same data for every repo.